### PR TITLE
Refactor helpers

### PR DESCRIPTION
**Pull request by ldevulder**
Should fix https://github.com/rancher/elemental/issues/918.

### CODE REVIEW

File: rancher/ioutil.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	if _, err = d.Write([]byte(data)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+/**
+ * Write data to file
+ * @remarks This function simply write data to a file
```
Comment (juadk) [inline rancher/ioutil.go:61]:
```suggestion
 * @remarks This function simply writes data to a file
```

File: rancher/ioutil.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	if _, err = d.Write([]byte(data)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+/**
+ * Copy file
+ * @remarks This function simply copy a file to another
```
Comment (juadk) [inline rancher/ioutil.go:84]:
```suggestion
 * @remarks This function simply copies a file to another
```

**Review by juadk — APPROVED** [review]

File: rancher/ioutil.go (hunk 1 above)
Comment (ldevulder) [reply rancher/ioutil.go]: Fixed, thanks!

File: rancher/ioutil.go (hunk 2 above)
Comment (ldevulder) [reply rancher/ioutil.go]: Fixed, thanks!

**Pull request merged**